src/itemset.js

```diff
@@ -115,11 +115,13 @@ define([
                     var possibleSrcs = _.map(datasources.getPossibleFixtures(), 
                                              function(val) { return val.src; }),
                         notCustom = _.contains(possibleSrcs, itemsetData.instance.src),
-                        choices = datasources.autocompleteChoices(itemsetData.instance.src);
+                        choices = datasources.autocompleteChoices(itemsetData.instance.src),
+                        strippedValue = itemsetData.valueRef.replace(/(\[.+])+/, ""),
+                        strippedLabel = itemsetData.labelRef.replace(/(\[.+])+/, "");
```

Comment:
Nope. That won't work. It only works for the end filters.

I don't think we care about that as users that do that sort of thing should know what they are doing